Aptoide/aptoide-client-v8
# [ASV-121] Changed Recover and Show password buttons lifecycle.

**What does this PR do?**

 Make possible for the user to use the recover and show password buttons after the view enters the Pause lifecycle.

**Database changed?**

No

**Where should the reviewer start?**

LoginSignUpCredentialsPresenter.java

**How should this be manually tested?**

Check if buttons are clickable after making the view go trough the different lifecycle (go to a web page, make view go to the background)

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/ASV-121

**Code Review Checklist**
